src/ctx.ts

```diff
@@ -0,0 +1,24 @@
+import { ButtonHandler } from "./ButtonHandler";
+import { Collection } from "discord.js";
+import { Command } from "./Command";
+
+class Ctx {
+	public readonly defaultGuild;
+	public readonly commands: Collection<string, Command> = new Collection<string, Command>();
+	public readonly buttons: Collection<string, ButtonHandler> = new Collection<string, ButtonHandler>();
```

Comment:
Why collections instead of maps, do we want to use any of the extended functionality?